slack/README.md

```diff
@@ -0,0 +1,62 @@
+# ilib-loctool-samples/slack
+
+This project contains a sample of a project that has slack
+mrkdwn strings encoded in a js file. Each file exports a
+simple javascript object with the following format:
+
+```
+    // comment for the following string
+    "unique-id": "a string in slack mrkdwn syntax"
+```
+
+The markdown syntax is hidden from the translators using
+xml-like components that are number sequentially as they
+are found in the source string. So, the string
+`This is _a *mrkdwn* string_ with a <http://slack.com|link> in it.` 
+will become 
+`This is <c0>a <c1>mrkdwn</c1> string</c0> with a <c2>link</c2> in it.`
+
+Notice that the link syntax is now hidden from the translator
+so they are not able to do things like localize the URL or
+screw up the mrkdwn syntax which they might not be familiar with.
+This assumes of course that translators are at least basically
+familiar with xml, which is usually the case, but not familiar
+with mrkdwn, which is a very niche syntax.
+
+Upon localization, the components will be recomposed back into 
```

Comment:
This repo nicely shows how XLIFF files are built from JavaScript objects. Could we also show how the components recomposition works?

Reply:
It shows it in the output files `translations/de/DE/strings.js` and `translations/ko/KR/strings.js`

Reply:
or did you mean something else?